core/src/main/java/com/netease/arctic/table/TableProperties.java

```diff
@@ -180,14 +180,20 @@ private TableProperties() {
    * table write related properties
    */
   public static final String BASE_FILE_FORMAT = "base.write.format";
-  public static final String BASE_FILE_FORMAT_DEFAULT = "parquet";
+  public static final String BASE_FILE_FORMAT_PARQUET = "parquet";
```

Comment:
It seems unnecessary to declare multiple variables for `parquet` and `orc` here. Can we directly use `org.apache.iceberg.FileFormat` or declare only once?

Reply:
> It seems unnecessary to declare multiple variables for `parquet` and `orc` here. Can we directly use `org.apache.iceberg.FileFormat` or declare only once?

Done, remove parquet and orc properity in TableProperties, use org.apache.iceberg.FileFormat instead.